src/sipjs-card.ts

```diff
@@ -416,6 +416,12 @@ class SipJsCard extends LitElement {
             case "scene":
                 service = "turn_on";
                 break;
+            case "light":
+                service = "toggle";
```

Comment:
This would probably work, but maybe it should be `turn_on`? You can't see the state of the entity, so how do you know if you turn it on or off?

Reply:
good ppoint but i think turn off would also sometimes be needed. Tha't's why i use toggle istead.

Reply:
But what if for example you want to unlock the door, but instead you lock it. I think it would only be frustrating if people exidently turn off a switch. It's not constant.

Maybe a option to select on or off?